### PR TITLE
custom work

### PR DESCRIPTION
**Pull request by manishiitg**
#### Reference issue
Fixes #...

### CODE REVIEW

File: client.go [hunk 1]
```diff
@@ -329,13 +330,13 @@ func (c *Client) OnNegotiationNeeded() {
 	// 1. pub create offer
 	offer, err := c.pub.pc.CreateOffer(nil)
 	if err != nil {
-		log.Errorf("id=%v err=%v", c.uid, err)
+		log.Debugf("id=%v err=%v", c.uid, err)
```
Comment (adwpc) [inline client.go:333]: I think log.Errorf is better, it's high level log

**Comment by leewardbound** [comment]
I think it's a bad idea to commit so many large movie files to this repo. Please find a way to make this PR not include large, unnecessary video files, it will greatly increase the library size; alternatively, since this is just an "example", you could move the code and all the videos to an unrelated repository.

File: example/simple/main.go [hunk 2]
```diff
@@ -74,14 +74,14 @@ func main() {
...
-	c, err := sdk.NewClient(e, addr, "client id")
+	c, err := sdk.NewClient(e, addr, "clientid2")
 	if err != nil {
 		log.Errorf("err=%v", err)
 		return
 	}
 	// subscribe rtp from sessoin
 	// comment this if you don't need save to file
-	c.OnTrack = saveToDisk
+	// c.OnTrack = saveToDisk
```
Comment (leewardbound) [inline example/simple/main.go:84]: I think this shouldn't be commented out in this example :)

File: example/track-to-rtp/README.md [hunk 3]
```diff
@@ -22,8 +22,11 @@ ffplay -protocol_whitelist file,udp,rtp -i subscribe.sdp
 
 broadcast to RTMP:
 ```
+
 # set your STREAM_KEY and RTMP_URL
-export RTMP_URL=rtmp://live.twitch.tv/app/$STREAM_KEY
+export RTMP_URL=rtmp://bom01.contribute.live-video.net/app/live_666332364_5791UvimKkDZW8edq8DAi4011wc4cR
```
Comment (leewardbound) [inline example/track-to-rtp/README.md:27]: Did you intend to publish your stream key here?

File: example/track-to-rtp/main.go [hunk 4]
```diff
@@ -61,110 +96,83 @@ func trackToRTP(track *webrtc.TrackRemote, receiver *webrtc.RTPReceiver) {
...
+				panic(err)
+			}
 
-	// create a new client from engine
-	c, err := sdk.NewClient(e, addr, "client id")
-	if err != nil {
-		log.Errorf("err=%v", err)
-		return
+			// Write
+			if _, err = c.conn.Write(b[:n]); err != nil {
```
Comment (leewardbound) [inline example/track-to-rtp/main.go:149]: When I originally wrote this sample, I also copied this same block of code from `pion/webrtc/examples/rtp-forwarder` but that example is based off a JSFiddle approach and a CLI interaction - I don't believe this block of code belongs in this example anymore.

File: example/ion-sfu-cluster-load-tool/README.md [hunk 5]
```diff
@@ -0,0 +1,80 @@
+## Ion-sfu load testing tool
```
Comment (leewardbound) [inline example/ion-sfu-cluster-load-tool/README.md:1]: Please update the README to reflect the purpose of the example, it is not clear what this example does, or why it's different than the other load tool example.

File: example/mirror-sfu/README.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
+# Simple
```
Comment (leewardbound) [inline example/mirror-sfu/README.md:1]: Please update the README to reflect the purpose of the example, it is not clear what this example does, and the README is from a different example.

**Review by leewardbound — CHANGES_REQUESTED** [review]

**Comment by leewardbound** [comment]
Thanks for your interest in helping with `ion-sdk-go`! I don't mean to be harsh but - In general, this PR adds a lot of small changes that I think would be better if you split up into smaller PRs that were more related - like changing `log.Error` to `log.Debug` without a good explanation/justification. Maybe those kinds of changes are appropriate, but they don't belong in one-big-lump PR together!

In my humble opinion, each Example you submit (or modify) should be its own PR.

**Comment by manishiitg** [comment]
@leewardbound i am sorry, this PR was submitted my mistakes. i didn't release this got submitted

**Comment by leewardbound** [comment]
No worries, I am sorry if I came off tough, hope the feedback was helpful 😊

**Comment by manishiitg** [comment]
yes. i submitted a new PR. that is genuine :) hope that helps

**Pull request closed without merging**